Annotations/Uri Annotation/UriAnnotation/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Syncfusion.Compression.Base" version="26.2.5" targetFramework="net48" />
```

Comment:
Why we specifically mention the version in the package? Can this be "*"?

Reply:
I have addressed the provided feedback